apps/frontend/src/shared-research/ResearchOutputList.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
-import { SharedResearchPageBody } from '@asap-hub/react-components';
+import { SharedResearchList } from '@asap-hub/react-components';
 
 import { useResearchOutputs } from './state';
 import { usePaginationParams, usePagination } from '../hooks';
 
-interface SharedResearchListProps {
+interface ResearchOutputListProps {
   searchQuery?: string;
   filters?: Set<string>;
 }
 
-const SharedResearchList: React.FC<SharedResearchListProps> = ({
+const ResearchOutputList: React.FC<ResearchOutputListProps> = ({
```

Comment:
Wait I don't understand the system. You renamed most things to shared research, except here you named FROM shared research to research output?

Reply:
It matches the name of the file which hasn't changed. It's not perfect but it's better than it was.

Reply:
Ok but then regardless of this file, what is the system? :sweat_smile: Thought it was 'research outputs are dead, long live shared research' but seems it is not